Makefile

```diff
@@ -65,18 +65,15 @@ src :=  $(filter-out src/ariane_regfile.sv, $(wildcard src/*.sv))      \
         $(wildcard src/cache_subsystem/*.sv)                           \
         $(wildcard bootrom/*.sv)                                       \
         $(wildcard src/clint/*.sv)                                     \
-        $(wildcard src/axi_node/src/*.sv)                              \
+        $(filter-out src/axi_node/src/axi_node_wrap_with_slices.sv, $(wildcard src/axi_node/src/*.sv))      \
```

Comment:
What is the problem with the AXI slices?

Reply:
These modules are not needed by default. However vcs treats them as top-level modules, but is too dumb to allow top-level modules with unconnected interfaces. Solution: move to an optional list when needed, automatically put -v flag (as needed) in front of them.